### PR TITLE
added solution two reverse No. using bits

### PR DESCRIPTION
**Pull request by rcoder23**
Added solution two reverse 2 numbers  #2095

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Here , We can swap 2 Numbers using bit manipulation (XOR operations ) .
It's faster way to swap two variables .

### Checklist

- [Y ] I've read the contribution guidelines.
- [ Y] I've checked the issue list before deciding what to submit.
- [ Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#2095
(Write your answer here.)

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@rcoder23 Add sample input-output, time complexity and space complexity.

File: Java/bit_manipulation/swap.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+		b=a^b;
+		a=a^b;
+		System.out.println(a);
+		System.out.println(b);
+
+	}
+}
+
+
+Example No1:- Let a=5 and b=10
```
Comment (atarax665) [inline Java/bit_manipulation/swap.java]: Please go through some recently merged Java PRs to get a better idea of the standard Java Template that we use.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by HarshCasper** [comment]
@rcoder23 Are you still working on it?

**Comment by rcoder23** [comment]
@HarshCasper yes updated.

**Comment by rcoder23** [comment]
@HarshCasper How i can join organization.?

**Comment by HarshCasper** [comment]
Make README updates as well. 

@rcoder23 Which organization are you looking to join?

File: Java/bit_manipulation/swap.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+*/
+import java.util.Scanner;
+public class swap {
+
+	public static void main(String[] args) {
+		Scanner sc=new Scanner(System.in);
+		System.out.println("Enter No a=");
+		int a=sc.nextInt();
+		System.out.println("Enter No a=");
```
Comment (atarax665) [inline Java/bit_manipulation/swap.java]:
```suggestion
		System.out.println("Enter No b=");

```

File: Java/bit_manipulation/swap.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+		int b=sc.nextInt();
+		//swapping 
+		a=a^b;
```
Comment (atarax665) [inline Java/bit_manipulation/swap.java]: Move the logic part to a new function, the main function should only have the driver code.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Review by rcoder23 — COMMENTED** [review]
separate logic code and driver code

**Comment by HarshCasper** [comment]
Make the README.md updates as well

**Comment by atarax665** [comment]
Closing due to inactivity.

**Pull request closed without merging**